### PR TITLE
Add Awesome CFP (Call For Papers)

### PR DESCRIPTION
**Pull request by lauthieb**
[https://github.com/lauthieb/awesome-cfp](https://github.com/lauthieb/awesome-cfp)

**awesome-cfp** is a collection of awesome Call For Papers to never miss to speak anymore !
We all know it, for speaker it is difficult to find a list of available Call For Papers in our country. With this awesome list we will be able to check conferences, links, dates, location and CFP end date.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by koppor** [comment]
Disclaimer: I am not a maintainer of this awesome list. However, the maintainer asked to review incoming lists before an own list is accepted (refs https://github.com/sindresorhus/awesome/issues/1323#issuecomment-394661729). Therefore, I'll put some comments.

A quick check reveals that following items are currently not followed:

- [ ] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- [ ] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
- [ ] Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).

The focus of the list is not clear to me. What is the difference to https://github.com/softwaremill/it-cfp-list? Is the list about designing call for papers or about concrete venues? Is the name of the list really matching the topic?

**Comment by lauthieb** [comment]
Hello @koppor, thanks a lot for your reactivity and your answer.

I have just follow your instructions and add a [contribution guidelines](https://github.com/lauthieb/awesome-cfp/blob/master/CONTRIBUTING.md) + a [CC0 licence](https://github.com/lauthieb/awesome-cfp/blob/master/LICENSE.md) to my repository.

To answer your question, the difference with it-cfp-list is that these events are sorted by country. Indeed, we can not all necessarily lead conferences in various places and this is also because of the language barrier for some. 

Therefore, this project lists the events available for specific countries. Indeed, the title is not necessarily appropriate but it still concerns the Call for papers and I wanted it to include "awesome" in its name. I hope after 30 days, my repository will be added to this awesome list.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

File: readme.md [hunk 1]
```diff
@@ -577,7 +577,7 @@
 - [Visual Regression Testing](https://github.com/mojoaxel/awesome-regression-testing) - Ensures changes did not break the functionality or style.
 - [Theravada](https://github.com/johnjago/awesome-theravada) - Teachings from the Theravada Buddhist tradition.
 - [inspectIT](https://github.com/inspectit-labs/awesome-inspectit) - Open source Java app performance management tool.
-
+- [Call For Papers](https://github.com/lauthieb/awesome-cfp) - Call For Papers to never miss to speak anymore.
```
Comment (meitar) [inline readme.md]: Please link directly to the `#readme` section. I.e., change: https://github.com/lauthieb/awesome-cfp to https://github.com/lauthieb/awesome-cfp#readme

**Comment by iamchathu** [comment]
Your list is different than any other awesome list.

**Comment by lauthieb** [comment]
Ok thanks guys. I understanf, I will close this pull request ;)

**Pull request closed without merging**